graphene_sqlalchemy/tests/test_types.py

```diff
@@ -200,42 +330,34 @@ class _TestSQLAlchemyConnectionField(SQLAlchemyConnectionField):
 
 
 def test_default_connection_field_factory():
-    _registry = Registry()
```

Comment:
Why ? ;)

Reply:
I realized that was not needed because the global registry is reset between tests (see [`confest.py`](https://github.com/graphql-python/graphene-sqlalchemy/blob/8ad1f75639ced6b00688b63c1a7b4a6a1acdf5af/graphene_sqlalchemy/tests/conftest.py#L13)).